content/tutorials/manage-the-rippled-server/configuration/configure-advisory-deletion.md

```diff
@@ -0,0 +1,98 @@
+
+To configure advisory deletion with a daily schedule, perform the following steps:
+
+1. Enable `advisory_delete` in the `[node_db]` stanza of your `rippled`'s config file.
+
+        [node_db]
+        # Other settings unchanged ...
+      	online_delete=2000
+      	advisory_delete=1
+
+    - Set `advisory_delete` to `1` to run online deletion only when prompted. (Set it to `0` to run online deletion automatically as new ledger versions become available.)
+    - Set `online_delete` to the minimum number of ledger versions to keep after running online deletion. The server accumulates more history than this until online deletion runs.
+
+    {% include '_snippets/conf-file-location.md' %}<!--_ -->
+
+2. Test running the [can_delete method][] to prompt the server to run online deletion.
+
+    You can use the [`rippled` commandline interface](get-started-with-the-rippled-api.html#commandline) to run this command. For example:
+
+        $ rippled --conf=/etc/opt/ripple/rippled.cfg can_delete now
+
+    The response indicates the maximum ledger index that may the server may delete from its ledger store. For example, the following message indicates that ledger versions up to and including ledger index 43633667 are to be deleted:
+
+        {
+          "result": {
+            "can_delete": 43633667,
+            "status": "success"
+          }
+        }
+
+    The server only deletes those ledger versions if the number of _newer_ validated ledger versions it has is equal to or greater than the `online_delete` setting.
+
+3. Configure your `cron` daemon to run the `can_delete` method you tested in the previous step at a scheduled time.
+
+    Edit your `cron` configuration:
+
+        $ crontab -e
+
+    The following example sets the server to run deletion at 1:05 AM server time daily:
+
+        5 1 * * * rippled --conf /etc/opt/ripple/rippled.cfg can_delete now
+
+    Be sure that you schedule the command to run based on your server's configured time zone.
+
+    **Tip:** You do not need to schedule a `cron` job to run online deletion if you have `advisory_delete` disabled. In that case, `rippled` runs online deletion automatically when the server has approximately twice the number of ledgers to keep after deletion.
+
+4. Start (or restart) the `rippled` service.
+
+        $ sudo systemctl restart rippled
+
+5. Periodically check your server's `complete_ledgers` range using the [server_info method][] to confirm that ledgers are being deleted as scheduled.
+
+    The lowest ledger index in `complete_ledgers` should increase after online deletion.
+
+    Deletion may take several minutes to complete when it runs, depending on how busy your server is and how much history you delete at a time.
+
+## Troubleshooting
+
+If online deletion does not seem to be running after configuring it, try the following:
+
+- Check that the user who configured the `cron` job has permissions to run the `rippled` server as a commandline client.
```

Comment:
This troubleshooting section is great!

For this particular troubleshooting tip above - would it be useful to mention this in the step for creating the `cron` job? Or is it obvious to a user who would be performing that task? 

And just to be sure I understand why this needs to be true -- the user that created the `cron` job will be the user used to issue the `can_delete` method at the scheduled time.

Reply:
Regarding your second point, yes, that's correct.

For the first point, I think that should be fine because the previous step has you run `rippled`, so I would expect that to come up before that point anyway. But it's in the troubleshooting in case users skipped past that.